lib/src/map_events.dart

```diff
@@ -40,9 +40,13 @@ final class _MapEvents {
     return listenersMap.values.toList();
   }
 
-  _MapEvents({BinaryMessenger? binaryMessenger}) {
-    _channel = MethodChannel('com.mapbox.maps.flutter.map_events',
-        const StandardMethodCodec(), binaryMessenger);
+  _MapEvents({BinaryMessenger? binaryMessenger, String channelSuffix = ''}) {
```

Comment:
Additionally binary messenger should be a required parameter as well, since in all other places binary messenger is specified explicitly, having it as a requirement will help with consistency.